### PR TITLE
Rocks

### PR DESCRIPTION
**Pull request by arturo-seijas**
Port Docker OCI images to Rock OCI images

### CODE REVIEW

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@rocks-support
```
Comment (jdkandersson) [inline .github/workflows/integration_test.yaml:8]: do we get this merged before merging this one?

Comment (arturo-seijas) [reply .github/workflows/integration_test.yaml:8]: We will, yes

**Review by gregory-schiano — COMMENTED** [review]
Could you write a description ?
Otherwise looking good to me, I'll approve once the operator-workflow PR is merged

**Review by gregnebu — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by weiiwang01 — APPROVED** [review]

**Pull request merged**